index.js

```diff
@@ -128,6 +128,11 @@ export const warnings = Object.freeze({
     code: "obfuscated-code",
     i18n: "sast_warnings.obfuscated_code",
     severity: "Critical"
+  },
+  weakCrypto: {
+    code: "weak-crypto",
+    i18n: "sast_warnings.weak_crypto",
+    severity: "Information"
```

Comment:
Maybe we should add another property `experimental: true` here ? So other tools like CLI may use this to show an experimental note.